metricbeat/module/beat/state/_meta/data.json

```diff
@@ -0,0 +1,34 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "beat": {
+        "id": "1f0c187b-f2ef-4950-b9cc-dd6864b9191a",
+        "state": {
+            "management": {
+                "enabled": false
+            },
+            "module": {
+                "count": 3
+            },
+            "output": {
+                "name": "elasticsearch"
+            },
+            "queue": {
+                "name": "mem"
+            }
+        },
+        "type": "metricbeat"
+    },
+    "event": {
+        "dataset": "beat.state",
+        "duration": 115000,
+        "module": "beat"
+    },
+    "metricset": {
+        "name": "state"
+    },
+    "service": {
+        "address": "127.0.0.1:5066",
+        "name": "beat",
+        "type": "beat"
```

Comment:
This does not seem to be correct. This should be `metricbeat`. And name should probably be what we have in `beat.name` or skip it.

Reply:
Done in 64eff0c24.